src/containers/RegisterForm.tsx

```diff
@@ -1,9 +1,21 @@
 import { connect } from 'react-redux';
 import { withFormik } from 'formik';
+import * as Yup from 'yup';
 
 import RegisterForm from '../components/RegisterForm';
 import { registerAccount } from '../actions/registerActions';
 
+const RegisterSchema = Yup.object().shape({
+  firstName: Yup.string().max(20, 'Too Long').required('Required'),
+  lastName: Yup.string().max(20, 'Too Long').required('Required'),
+  email: Yup.string().email('Invalid Email').required('Required'),
+  password: Yup.string().min(9, 'Too Short').required('Required'),
```

Comment:
o lmao its 9 characters
Last nit: can we just specify ('Password must be at least 9 characters') / ('Password must be less than 21 characters')